src/client/pages/Bodem/Bodem.test.tsx

```diff
@@ -113,51 +155,149 @@ const testState: any = {
               id: '',
               datePublished: '2022-12-01T09:53:11Z',
               isActive: true,
-              isChecked: true,
+              isChecked: false,
             },
             {
               status: 'In behandeling',
               id: '',
-              datePublished: '',
-              isActive: false,
+              datePublished: '2022-12-01T09:54:11Z',
+              isActive: true,
               isChecked: false,
             },
             {
               status: 'Afgehandeld',
               id: '',
-              datePublished: '',
-              isActive: false,
-              isChecked: false,
+              datePublished: '2022-12-01T09:55:11Z',
+              isActive: true,
+              isChecked: true,
             },
           ],
         },
       ],
     },
     status: 'OK',
   },
-};
+} as unknown as AppState;
+
+const createComponent = componentCreator({
+  component: Bodem,
+  routeEntry: AppRoutes.BODEM,
+  routePath: AppRoutes.BODEM,
+});
 
 describe('Bodem', () => {
-  describe('with results', () => {
-    const renderBodem = setupMockApp(Bodem, 'BODEM', testState);
+  test('Title is present', () => {
+    const MockBodem = createComponent(testState);
+    const screen = render(<MockBodem />);
 
-    it('should show the right overviewpage', () => {
-      const { asFragment } = renderBodem();
+    const title = screen.getByRole('heading', { name: 'Bodem' });
+    expect(title).toBeInTheDocument();
 
-      expect(asFragment()).toMatchSnapshot();
-    });
+    const desc = screen.getByText(
+      /Op deze pagina vindt u informatie over uw lood in de bodem-check/
+    );
+    expect(desc).toBeInTheDocument();
   });
 
-  describe('without results', () => {
-    const renderBodem = setupMockApp(Bodem, 'BODEM', {
-      BODEM: {
-        content: {},
-      },
+  describe('Tables', () => {
+    const lopendePattern = /(In behandeling)|Ontvangen/;
+    const afgehandeldePattern = /Afgehandeld|Afgewezen/;
+
+    test('Table items found', () => {
+      const MockBodem = createComponent(testState);
+      const screen = render(<MockBodem />);
+
+      const lopendeAanvraagTableHeader = screen.getByRole('heading', {
+        name: 'Lopende aanvragen',
+      });
+
+      const afgehandeldeAanvraagTableHeader = screen.getByRole('heading', {
+        name: 'Afgehandelde aanvragen',
+      });
+
+      const columnHeaders = ['Adres', 'Aangevraagd', 'Status'];
+
+      const lopendeAanvraagTable = lopendeAanvraagTableHeader.parentElement!;
+      const lopendeAanvraagColumnHeaders =
+        within(lopendeAanvraagTable).getAllByRole('columnheader');
+      expect(
+        lopendeAanvraagColumnHeaders.map((header) => header.textContent)
+      ).toStrictEqual(columnHeaders);
+
+      const lopendeAanvraagRows = within(lopendeAanvraagTable).getAllByRole(
+        'row',
+        {
+          name: lopendePattern,
+        }
+      );
+      expect(lopendeAanvraagRows.length).toBe(2);
```

Comment:
Ik zou deze los van elkaar testen (Ontvangen/in Behandeling) en zonder regex. Wat denk jij? Je kan nu 2x Ontvangen terugkrijgen zonder dat de test dan faalt
```suggestion
      const lopendeAanvraagHasOntvangenRow = within(lopendeAanvraagTable).queryByRole(
        'row',
        {
          name: 'Ontvangen',
        }
      );
      - expect(lopendeAanvraagHasOntvangenRow).toBeInTheDocument();
      const lopendeAanvraagHasInBehandelingRow = within(lopendeAanvraagTable).queryByRole(
        'row',
        {
          name: 'In behandeling',
        }
      );
      expect(lopendeAanvraagHasInBehandelingRow).toBeInTheDocument();
```

Reply:
Ja ik vind het beter. Heb het iets anders gedaan, ik ben benieuwd of dat is wat je bedoelt